bedrock/mozorg/tests/test_views.py

```diff
@@ -350,14 +350,15 @@ def test_technology_locale_template(self, render_mock):
 
 @patch('bedrock.mozorg.views.l10n_utils.render')
 class TestHome(TestCase):
-    def test_home_enUS(self, render_mock):
+    def test_home_quantum_template(self, render_mock):
         request = RequestFactory().get('/')
         request.locale = 'en-US'
         views.home(request)
-        render_mock.assert_called_once_with(request, 'mozorg/home/home-new.html')
+        render_mock.assert_called_once_with(request, 'mozorg/home/home-quantum.html')
 
-    def test_home_non_enUS(self, render_mock):
+    @patch.object(views, 'lang_file_is_active', lambda *x: False)
```

Comment:
No idea if I did this right, I copied from elsewhere, but from context it seems like this makes the test behave as if the lang file is NOT active and loads the fallback template. Am I guessing right? It passes locally so I hope this works.

Reply:
Looks good to me 😄